dags/monitoring/events.py

```diff
@@ -0,0 +1,80 @@
+from datetime import datetime, timedelta
+
+from airflow import DAG
+from airflow.contrib.operators.kubernetes_pod_operator import KubernetesPodOperator
+from airflow.models import Variable
+from airflow.utils.trigger_rule import TriggerRule
+
+from dags.airflow_utils import MATTERMOST_DATAWAREHOUSE_IMAGE, pod_defaults, send_alert
+from dags.kube_secrets import (
+    SNOWFLAKE_ACCOUNT,
+    SNOWFLAKE_LOAD_DATABASE,
+    SNOWFLAKE_LOAD_ROLE,
+    SNOWFLAKE_LOAD_WAREHOUSE,
+    SNOWFLAKE_PASSWORD,
+    SNOWFLAKE_USER,
+)
+from plugins.operators.mattermost_operator import MattermostOperator
+
+# Default arguments for the DAG
+default_args = {
+    "depends_on_past": False,
+    "owner": "airflow",
+    "on_failure_callback": send_alert,
```

Comment:
where is the alert being sent?

Reply:
On the channel that has been configured. There's some space for improvement on the semantics of the `MattermostOperator`, but let's do it in a separate PR.